skeleton.go

```diff
@@ -594,10 +604,51 @@ func (s *skeleton) processAnalysisResultBatches(ctx context.Context) {
 	}
 }
 
-func (s *skeleton) cleanUpCerberusQueueItems(ctx context.Context) {
-	// Todo
-	//-- clean up old stuff
-	//delete from astm.sk_cerberus_queue_items where created_at < now()-interval '14 days';
+const limit = 5000
+
+func (s *skeleton) runCleanupJobs() {
+	s.cleanupCerberusQueueItems()
+	s.cleanupAnalysisResults()
```

Comment:
unused analysis requests can be deleted, thats fine, but im not sure if results and queue items should be deleted over time at all, without making backup

i'd personally prefer some kind of archive mechanism that would copy the old data to a different archive db before deleting, either manually, either automatically. 
imho manual backup once per (half)year would be enough to keep the db size reasonable, performance shouldn't be affected much due to these tables growing over time